site/survey.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Your Name">
+    <meta name="description" content="A personal website with information about me and my favorite things.">
+    <title>My Website</title>
+    
+</head>
+<body>
+
+    <div class="width-wrap">
+        <header id="top">
+            <nav>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="Links.html">Links</a></li>
+                    <li><a href="Glossary.html">Glossary</a></li>
+                </ul>
+            </nav>
+        </header>   
+        <main>
+            <h1>Survey</h1>
+    <form action="https://formspree.io/f/YOUR_ENDPOINT" method="POST">
```

Comment:
This should've had the actual endpoint and not a templated value of `YOUR_ENDPOINT`
```suggestion
    <form action="https://formspree.io/f/YOUR_ENDPOINT" method="POST">
```